presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestNativeTpchConnectorQueries.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import org.testng.annotations.Test;
+
+public abstract class AbstractTestNativeTpchConnectorQueries
+        extends AbstractTestQueryFramework
+{
+    @Test
+    public void testTpchTinyTables()
```

Comment:
use a data provider to break this up into tests that an pass and fail independently.